.do/values-env-temp.yaml

```diff
@@ -0,0 +1,34 @@
+# environment specific values
+global:
+  github:
+    app:
+      name: ${GITHUB_APP_NAME}
+      id: "${GITHUB_APP_ID}"
+      privateKey: "${GITHUB_PRIVATE_KEY}"
+    clientId: ${GITHUB_CLIENTID}
+    clientSecret: ${GITHUB_CLIENTSECRET}
```

Comment:
Can we split the word CLIENTSECRET to CLIENT_SECRET? Likewise with CLIENTID -> CLIENT_ID. This applies throughout the file, there are a few cases of it not being consistent :)

Reply:
I'm just refactoring how these variables are passed to helm, using a values file and some standards form variable naming, instead of having repeated code and huge lines with all variables mapping. Maybe the actual naming of the variables can be changed in a subsequent step.